components/Editor.js

```diff
@@ -251,15 +262,24 @@ class Editor extends React.Component {
               resetDefaultSettings={this.resetDefaultSettings}
             />
             <div className="buttons">
-              {this.props.tweet && this.state.online && (
-                <Button
-                  className="tweetButton"
-                  onClick={this.upload}
-                  title={this.state.uploading ? 'Loading...' : 'Tweet Image'}
-                  color="#57b5f9"
-                  style={{ marginRight: '8px' }}
-                />
-              )}
+              <Button
+                className="tweetButton"
+                onClick={() => this.format(this.state.code)}
+                title={this.state.formatLoading ? 'Formatting...' : 'P'}
```

Comment:
Remove `state.formatLoading` since formatting is synchronous.

Reply:
Done.